clifford/_multivector.py

```diff
@@ -13,30 +13,27 @@
 
```

Comment:
See https://external-builds.readthedocs.io/html/clifford/166/generated/clifford.MultiVector.html#clifford.MultiVector for the rendered version of this file.